Engineering/BestPractices/AgileManifesto.md

```diff
@@ -0,0 +1,39 @@
+# Learn about Agile Manifesto and try to follow its values and principles
+
+## (https://agilemanifesto.org)
+
+### We are uncovering better ways of developing software by doing it and helping others do it. Through this work we have come to value:
+
+* Individuals and interactions over processes and tools
+
+* Working software over comprehensive documentation
+
+* Customer collaboration over contract negotiation
+
+* Responding to change over following a plan
```

Comment:
The Agile Manifesto has a copyright notice that we need to comply with, "© 2001, the above authors this declaration may be freely copied in any form, but only in its entirety through this notice." The above needs to include everything from http://agilemanifesto.org/, through the copyright notice, or just link to http://agilemanifesto.org/.